checks/checkforcontent.go

```diff
@@ -68,20 +68,24 @@ func CheckFilesContent(checkName, shellPathFnPattern string,
 	onFileContent func(path string, content []byte,
 		Logf func(s string, f ...interface{})) (bool, error),
 ) checker.CheckResult {
-	predicate := func(filepath string) bool {
+	predicate := func(filepath string) (bool, error) {
 		// Filter out Scorecard's own test files.
 		if isScorecardTestFile(c.Owner, c.Repo, filepath) {
-			return false
+			return false, nil
 		}
 		// Filter out files based on path/names using the pattern.
 		b, err := isMatchingPath(shellPathFnPattern, filepath, caseSensitive)
 		if err != nil {
-			return false
+			return false, err
 		}
-		return b
+		return b, nil
 	}
 	res := true
-	for _, file := range c.RepoClient.ListFiles(predicate) {
```

Comment:
I think `CheckFilesContent()` is no longer used. I have not removed it yet as part of post-migration cleanup. Feel free to remove it.

Reply:
Done.